hs_core/templates/pages/genericresource.html

```diff
@@ -975,6 +989,78 @@ <h4 class="modal-title">Confirm file deletion</h4>
                                 </div>
                                 {% endif %}
 
+                                  <!-- ========= CONNECTIONS TAB ========= -->
+                                {% if connections %}
+                                    <div class="tab-pane" id="connectionsTab">
+                                         {% if connections.collected_by %}
+                                                <h4><strong>Belongs to Collections:</strong></h4>
+                                                <hr style="margin-top: 0;margin-bottom: 2px">
+                                                <table id="collected-by-table" class="table-hover table-striped resource-custom-table" width="100%">
+                                                    <thead>
+                                                        <tr>
+                                                            <th>Title</th>
+                                                            <th>Owners</th>
+                                                            <th>Sharing Status</th>
+                                                            <th>My Permission</th>
+                                                        </tr>
+                                                    </thead>
+                                                    <tbody>
+                                                        {% for res in connections.collected_by %}
```

Comment:
@zhiyuli - this looks good. One minor suggestion - instead of "Belongs to Collections" as the header I think it should read "This resource belongs to the following collections:"